app-modules/ai/resources/views/components/assistant.blade.php

```diff
@@ -459,6 +459,24 @@ class="ml-auto flex h-6 w-6 cursor-pointer items-center gap-2 rounded-md p-1 tex
                     <form x-on:submit.prevent="sendMessage">
                         <div
                             class="w-full overflow-hidden rounded-xl border border-gray-950/5 bg-gray-50 shadow-sm dark:border-white/10 dark:bg-gray-700">
+                            @if ($this->files)
+                                <div class="flex flex-row justify-start px-4 py-2">
+                                    @foreach ($this->files as $key => $file)
+                                        <div class="mx-2 flex items-center">
+                                            <x-filament::badge>
+                                                {{ $file['name'] }}
+                                            </x-filament::badge>
+                                            <x-filament::icon-button
+                                                aria-label="Remove uploaded file {{ $file['name'] }}"
+                                                size="xs"
+                                                icon="heroicon-o-x-mark"
+                                                wire:click="removeUploadedFile({{ $key }})"
+                                            >
+                                                </x-filament::button>
```

Comment:
This could be refactored to use the `deleteButton` feature as a slot inside the badge, which is the same UI that we use for the TagsInput

```blade
<x-slot
    name="deleteButton"
    label="Remove uploaded file {{ $file['name'] }}"
    wire:click="removeUploadedFile({{ $key }})"
></x-slot>
```